newfeature.js

```diff
@@ -0,0 +1,60 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable no-use-before-define */
+/* eslint-disable max-classes-per-file */
```

Comment:
- [ ] Kindly please take note that according to best practices, we need to try our possible best not to disable any of the linters.
- [ ] For line 1, kindly look through your codebase and remove any unused variable. 💁‍♂️ 
- [ ] For line 2, you may have to define your variable before calling it. 💁‍♂️ 
- [ ] For line 3, possibly place the two classes you have in your app in two different files; perhaps `book.js` and `bookscreen.js.` 👍